mne/viz/tests/test_misc.py

```diff
@@ -165,4 +166,12 @@ def test_plot_dipole_amplitudes():
     dipoles.plot_amplitudes(show=False)
 
 
+def test_plot_csd():
+    """Test plotting of CSD matrices."""
+    csd = CrossSpectralDensity([1, 2, 3], ['CH1', 'CH2'], tmin=0, tmax=1,
+                               frequencies=[(10, 20)], n_fft=1)
+    plot_csd(csd, mode='csd')  # Plot cross-spectral density
+    plot_csd(csd, mode='coh')  # Plot coherence
```

Comment:
please add `plt.close('all')` at the end to prevent too many mpl figures (feel free to add it to `test_plot_dipole_amplitudes` while you're in there)